webhooks-extension/pkg/endpoints/webhook.go

```diff
@@ -66,6 +66,9 @@ func (r Resource) createWebhook(request *restful.Request, response *restful.Resp
 	}
 
 	dockerRegDefault := r.Defaults.DockerRegistry
+	// remove prefixes if any
+	webhook.DockerRegistry = strings.TrimPrefix(webhook.DockerRegistry, "https://")
```

Comment:
This is a bit funky, we're stripping things here but in the getting started doc we have the prefix - so what's the deal with what's accepted and what isn't?

Reply:
so in theory we don't accept registries with `http` or `https` but, by stripping them in the back end, we can let the end user add whichever prefixes they want. Should i clarify this in the doc?